snapcraft/cli/_errors.py

```diff
@@ -113,7 +114,7 @@ def exception_handler(  # noqa: C901
     is_snapcraft_reportable_error = issubclass(
         exception_type, errors.SnapcraftReportableError
     )
-    is_raven_setup = RavenClient is not None
+    is_raven_setup = RavenClient is not None if is_snap() else False
```

Comment:
this should be is_snap, the is_raven_setup var could be replaced with `is_snap()` completely as the snap will always have raven.

Reply:
Correct. Changing to is_snap().